### PR TITLE
Cleanup Kruize directory structure

### PR DESCRIPTION
**Pull request by dinogun**
This PR does some general cleanup related to naming conventions and directory structure
1. Rename most `autotune` instances to `kruize`
2. Rename `AutotuneConfig` to `kruizeLayer`
3. Rename `dbactivites` to `database`
4. Rename `KruizeDeployment` to `KruizeOperator` and move it to `com.autotune.operator` dir
5. Move performanceProfiles to `analyzer` module (`analyzer/performanceProfiles` dir)
6. Move all recommendation related classes to `analyzer/recommendations` dir
7. Move recommendation algorithms to analyzer/recommendations/algos dir
8. Move all experiment related classes to `com.autotune.analyzer.experiment`
9. Rename `com.autotune.common.experiments` to `com.autotune.common.trials`
10. Rename `validationResultData` to `validationOutputData`
11. Remove all files related to `src/main/java/com/autotune/common/queue`
12. Move `AnalyzerConstants` and `AnalyzerErrorConstants` to `analyzer` module
13. Move all classes related to KruizeObject to analyzer/kruizeObject dir
14. Move all classes related to layers to analyzer/kruizeLayer dir
15. Move all classes related to experiment to analyzer/experiment dir
16. Move analyzer/variables to common/variables

### CODE REVIEW

**Review by bharathappali — APPROVED** [review]
Based on the discussion over the call about clean up and structuring the code base, It Looks Good To Me!

**Comment by bharathappali** [comment]
Thanks @dinogun for raising this PR, It helps to fix the issue #660

File: src/main/java/com/autotune/analyzer/Analyzer.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ public static void addServlets(ServletContextHandler context) {
         context.addServlet(ListStacks.class, ServerContext.LIST_STACKS);
         context.addServlet(ListStackLayers.class, ServerContext.LIST_STACK_LAYERS);
         context.addServlet(ListStackTunables.class, ServerContext.LIST_STACK_TUNABLES);
-        context.addServlet(ListAutotuneTunables.class, ServerContext.LIST_AUTOTUNE_TUNABLES);
+        context.addServlet(ListKruizeTunables.class, ServerContext.LIST_AUTOTUNE_TUNABLES);
```
Comment (chandrams) [inline src/main/java/com/autotune/analyzer/Analyzer.java:55]: @dinogun - I think this change needs to be reflected in some tests, API design doc and kruize-demos (autotune minikube script). We can make these changes in separate PRs. Approving this PR, Thank you.

**Review by chandrams — APPROVED** [review]
LGTM

**Comment by khansaad** [comment]
LGTM!
Will take this to complete the container refactoring  in #637

File: src/main/java/com/autotune/analyzer/Analyzer.java (hunk 1 above)
Comment (dinogun) [reply src/main/java/com/autotune/analyzer/Analyzer.java]: tests and API docs now updates. Will raise a PR for kruize-demos
